From 5d2ec4d8ecd7ab6bd4a62bec6b2f63577549b3bd Mon Sep 17 00:00:00 2001
Message-Id: <5d2ec4d8ecd7ab6bd4a62bec6b2f63577549b3bd.1347457394.git.minovotn@redhat.com>
From: Michal Novotny <minovotn@redhat.com>
Date: Wed, 18 Jul 2012 14:03:18 +0200
Subject: [PATCH 1/6] qemu-options.hx: Improve -{read|write}config options
 description

RH-Author: Michal Novotny <minovotn@redhat.com>
Message-id: <f892485319ed7e3176164b531ee3ac212f53ca81.1342619852.git.minovotn@redhat.com>
Patchwork-id: 40371
O-Subject: [RHEL-6.4 qemu-kvm PATCH v2] qemu-options.hx: Improve -{read|write}config options description
Bugzilla: 818134
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Pavel Hrdina <phrdina@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

Bugzilla: 818134
Upstream relationship: ed24cfac07289d458ff284898bbd0389743a18b7

This is the patch to improve description for -{read|write}config
functions.

Signed-off-by: Michal Novotny <minovotn@redhat.com>
Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
---
 qemu-options.hx |   13 ++++++++++++-
 1 file changed, 12 insertions(+), 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-options.hx | 13 ++++++++++++-
 1 file changed, 12 insertions(+), 1 deletion(-)

diff --git a/qemu-options.hx b/qemu-options.hx
index 1b20237..27e3d51 100644
--- a/qemu-options.hx
+++ b/qemu-options.hx
@@ -2155,10 +2155,21 @@ DEF("old-param", 0, QEMU_OPTION_old_param,
 #endif
 DEF("readconfig", HAS_ARG, QEMU_OPTION_readconfig,
     "-readconfig <file>\n")
+STEXI
+@item -readconfig @var{file}
+Read device configuration from @var{file}. This approach is useful when you want to spawn
+QEMU process with many command line options but you don't want to exceed the command line
+character limit.
+ETEXI
 DEF("writeconfig", HAS_ARG, QEMU_OPTION_writeconfig,
     "-writeconfig <file>\n"
     "                read/write config file\n")
-
+STEXI
+@item -writeconfig @var{file}
+Write device configuration to @var{file}. The @var{file} can be either filename to save
+command line and device configuration into file or dash @code{-}) character to print the
+output to stdout. This can be later used as input file for @code{-readconfig} option.
+ETEXI
 DEF("no-kvm", 0, QEMU_OPTION_no_kvm,
     "-no-kvm         disable KVM hardware virtualization\n")
 DEF("no-kvm-irqchip", 0, QEMU_OPTION_no_kvm_irqchip,
-- 
1.7.11.4

